garage/tf/models/gaussian_mlp_model.py

```diff
@@ -0,0 +1,206 @@
+"""GaussianMLPModel."""
+import numpy as np
+import tensorflow as tf
+
+from garage.misc import ext
+from garage.tf.core.mlp import mlp
+from garage.tf.core.parameter import parameter
+from garage.tf.models.base import TfModel
+
+
+class GaussianMLPModel(TfModel):
+    """
+    GaussianMLPModel.
+
+    Args:
+    :param output_dim: Output dimension of the model.
+    :param name: Name of the model.
+    :param hidden_sizes: List of sizes for the fully-connected hidden
+        layers.
+    :param learn_std: Is std trainable.
+    :param init_std: Initial value for std.
+    :param adaptive_std: Is std a neural network. If False, it will be a
+        parameter.
+    :param std_share_network: Boolean for whether mean and std share the same
+        network.
+    :param std_hidden_sizes: List of sizes for the fully-connected layers
+        for std.
+    :param min_std: Whether to make sure that the std is at least some
+        threshold value, to avoid numerical issues.
+    :param max_std: Whether to make sure that the std is at most some
+        threshold value, to avoid numerical issues.
+    :param std_hidden_nonlinearity: Nonlinearity for each hidden layer in
+        the std network.
+    :param std_output_nonlinearity: Nonlinearity for output layer in
+        the std network.
+    :param std_parametrization: How the std should be parametrized. There
+        are a few options:
+        - exp: the logarithm of the std will be stored, and applied a
+            exponential transformation
+        - softplus: the std will be computed as log(1+exp(x))
+    :param hidden_nonlinearity: Nonlinearity used for each hidden layer.
+    :param output_nonlinearity: Nonlinearity for the output layer.
+    """
+
+    def __init__(self,
+                 output_dim,
+                 name=None,
+                 hidden_sizes=(32, 32),
+                 learn_std=True,
+                 init_std=1.0,
+                 adaptive_std=False,
+                 std_share_network=False,
+                 std_hidden_sizes=(32, 32),
+                 min_std=1e-6,
+                 max_std=None,
```

Comment:
Please group `init_std, min_std, max_std` together, 
and `learn_std, adaptive_std, std_share_network` together. 
and `hidden_sizes, hidden_nonlinearity, output_nonlinearity` together 
and `std_hidden_sizes, std_hidden_nonlinearity, std_output_nonlinearity` together.